src/browser/browser.zig

```diff
@@ -158,8 +158,17 @@ pub const Session = struct {
         }
     }
 
-    pub fn createPage(self: *Session) !Page {
-        return Page.init(self.alloc, self);
+    pub fn createPage(self: *Session) !void {
```

Comment:
What about returning the `*Page` pointer? instead of have many `browser.session.page().XXX`